### ISSUE TITLE
[Suggestion] Add support for wsrv.nl image proxy.

### ISSUE THREAD

**Issue by Hellscaped** [divyam234/teldrive#113]
wsrv.nl is an open source image proxy, it can resize items easily.
It is open source (https://github.com/weserv/images) and is extremely easy to use. I would implement this myself as it would be pretty easy to implement (a few changes in url arguments), but I'm really bad at go.

**Comment by divyam234** [comment]
@Hellscaped  Curent resizer works very well its alos  uses libvips underhood

**Comment by Hellscaped** [comment]
> @Hellscaped Curent resizer service works very well its also uses libvips underhood

Yes, but there are many wesrv public instances that one can use instead of running their own of your implementation.

**Comment by divyam234** [comment]
That is true but it will not work on local instance of teldrive  if teldrive is deployed on public server then it could work

**Comment by jontstaz** [comment]
> @Hellscaped Curent resizer service works very well its also uses libvips underhood

Which service currently works with teldrive? I was trying to find what to put in the `Image Resizer Host` but couldn't see any info.
